docs/_tutorials/03_extending_wash/writing-an-external-plugin.md

```diff
@@ -94,7 +94,7 @@ wash . ❯ ls
 aws        docker     gcp        kubernetes local_fs
 ```
 
-If you try to `ls local_fs`, you should get an error
+We're not done yet, so if you try to `ls local_fs`, you'll get this error:
 
 ```
 wash . ❯ ls local_fs
```

Comment:
The repetition in the output is pretty annoying. `list` doesn't do that. I'm not sure why `ls` seems to retry a bunch before failing.

Reply:
Yeah, I'm not sure either. Should fix that once https://github.com/puppetlabs/wash/issues/284 is done.